tribomobile/components/CheckBoxCustom.js

```diff
@@ -1,25 +1,30 @@
 import React, {Component} from 'react';
 import {View, Text, StyleSheet} from 'react-native';
-import { CheckBox } from 'native-base';
-import Colors from "../src/Colors";
+import CheckBox from '@react-native-community/checkbox';
+import Colors from '../src/Colors';
 
-function CheckboxCustom ({title,value}) {
+function CheckboxCustom({customStyle, onChange, title, value}) {
   return (
-    <View style={styles.checkBoxView}>
-      <CheckBox checked={value} color={Colors.BlueLight}/>
-      <Text style={{marginLeft: 15}}>{title}</Text>
+    <View style={[customStyle, styles.checkBoxView]}>
+      {/* <CheckBox checked={value} color={Colors.BlueLight}/>
```

Comment:
Delete the empty object prior to merge, unless it is usable later.

Reply:
Thank you for noticing it! I will probably use it, nice job